software/password-manager.yml

```diff
@@ -0,0 +1,11 @@
+name: "Laravel Simple Password Manager"
+website_url: "https://github.com/milenmk/Laravel-Simple-Password-Manager"
+source_code_url: "https://github.com/milenmk/Laravel-Simple-Password-Manager"
+description: "Password manager written with PHP and Laravel.."
+licenses:
+  - GPL-3.0
+platforms:
+  - PHP
+tags:
+  - Password Managers
+depends_3rdparty: true
```

Comment:
What third-party service does it depend on?

Reply:
I wasn't sure what to put here in the beginning. I've change it to false now. In reality, the software doesn't connect to any  API or website or any of this sort to work. On the other hand, it doesn't have a Docker image, so the prerequizite is to have LAMP stack or similar previously installed.

Reply:
Feel free to omit it if it doesn't connect to third-party services, that line is if it depends on like Google, etc.